ansible/roles/etcd/handlers/main.yml

```diff
@@ -4,11 +4,9 @@
 
 - name: restart etcd
```

Comment:
No longer needed if the ``restart etcd`` is triggered at the end of ``tasks/main.yml``